### PR TITLE
Fixing issues with Slack notifications, missing/wrong variables

### PR DESCRIPTION
**Pull request by AntonioKL**

### CODE REVIEW

**Comment by AntonioKL** [comment]
Adding feature to allow manage subscription out of admin GUI

File: app/Http/Middleware/SubscribersConfigured.php [hunk 1]
```diff
@@ -52,10 +52,6 @@ public function __construct(Repository $config)
      */
     public function handle(Request $request, Closure $next)
     {
-        if (!$this->config->get('setting.enable_subscribers')) {
```
Comment (jbrooksuk) [inline app/Http/Middleware/SubscribersConfigured.php]: Why have we removed this condition?

Comment (AntonioKL) [reply app/Http/Middleware/SubscribersConfigured.php]: > Why have we removed this condition?

The subscription page must be manageable, despite the setting "Allow to subscribe". Admin should always be able to configure the subscribers and manage their subscriptions, etc. 

You can not have this setting enabled or not based on "if the user is allowed to subscribe or not feature"

Comment (jbrooksuk) [reply app/Http/Middleware/SubscribersConfigured.php]: Then the condition should check for an active admin user, then?

Comment (AntonioKL) [reply app/Http/Middleware/SubscribersConfigured.php]: It is done at the auth module ( outside of the scope for this commit)

Comment (jbrooksuk) [reply app/Http/Middleware/SubscribersConfigured.php]: Why can't we add a condition into https://github.com/CachetHQ/Cachet/blob/2.4/app/Http/Middleware/SubscribersConfigured.php for admin?

Comment (AntonioKL) [reply app/Http/Middleware/SubscribersConfigured.php]: It is handled in app/Http/Kernel.php 
@jbrooksuk

Comment (jbrooksuk) [reply app/Http/Middleware/SubscribersConfigured.php]: I don't understand?

Comment (jbrooksuk) [reply app/Http/Middleware/SubscribersConfigured.php]: If subscriptions aren't enabled and you're **not** an admin, then you shouldn't be able to see the subscription page, so as far as I can tell, the middleware is required.

**Comment by jbrooksuk** [comment]
@AntonioKL if you could address the middleware issue above, we could get this merged 😄

**Comment by AntonioKL** [comment]
@jbrooksuk

Please merge the code. Just tested the code and middleware in place

**Comment by li-olivierfostier** [comment]
Hello,
Can some tell us where/how can we adjust parameters to add Slack notification ?
Nothing found in the documentation.

Thx

**Comment by Pimorez** [comment]
> 
> 
> Hello,
> Can some tell us where/how can we adjust parameters to add Slack notification ?
> Nothing found in the documentation.
> 
> Thx

For now you have to manually update the `subscribers` table in the database. It contains a slack_webhook_url column

**Comment by jbrooksuk** [comment]
Thank you for your input on Cachet 2.x. We are shifting our attention and resources to Cachet 3.x and will no longer be supporting the 2.x version. If your feedback or issue is relevant to the 3.x series, we encourage you to engage with the new branch.

For more information on the Cachet rebuild and our plans for 3.x, you can read the announcement [here](https://github.com/CachetHQ/Cachet/discussions/4342).

We appreciate your understanding and look forward to your contributions to the new version.

**Pull request closed without merging**